build.gradle

```diff
@@ -40,7 +40,7 @@ repositories
     // For Spark CDH
     maven { url "https://repository.cloudera.com/content/repositories/releases/" }
     // For jetty (through spark)
-    maven { url "http://repo.spring.io/plugins-release/" }
+    maven { url "http://repository.cloudera.com/cloudera/cloudera-repos/" }
```

Comment:
What necessitated this particular change?

Reply:
Change in the dependent location causing the build to fail.